src/web/services/table-comparator.service.ts

```diff
@@ -11,6 +11,27 @@ export class TableComparatorService {
 
   constructor() { }
 
+  /**
+   * Compares two numbers naturally depending on order given.
+   */
+  compareAsNumber(order: SortOrder, numA: number, numB: number): number {
```

Comment:
My indentation is actually suggest you to use `compare` in the service. The comparison logic here is essentially the same as `compareNaturally`.

Reply:
Initially thought comparing number as string might give unintended behaviour. Did not realise it is handled by option in `.localeCompare()`. 

Updated.